tests/phpunit/includes/utils.php

```diff
@@ -465,7 +465,7 @@ function test_rest_expand_compact_links( $links ) {
 	}
 	foreach ( $links as $rel => $links_array ) {
 		if ( ! strpos( $rel, ':' ) ) {
-			continue;
+			break;
```

Comment:
This `break;` is in the wrong place.

@HongPong @mattyrob this is pretty difficult to keep up with; I'd suggest re-creating this PR using `git` to pull the patches over.  See https://github.com/ClassicPress/ClassicPress/issues/263#issuecomment-446473885 and the following comment for more specific info.